prow/deploy.sh

```diff
@@ -29,6 +29,11 @@ set -o pipefail
 if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
```

Comment:
this seems like a weak proxy for human or not, considering this is a standard env.
we could set an env on the pod like `SKIP_CONFIRMATION = true`

Reply:
@ibzib you are actually a robot :robot:

Reply:
ideally we can `gcloud auth list` and verify it's not a robot account?